lms/envs/aws.py

```diff
@@ -581,6 +581,8 @@
             'schedule': datetime.timedelta(hours=ENV_TOKENS.get('THIRD_PARTY_AUTH_SAML_FETCH_PERIOD_HOURS', 24)),
         }
 
+    SOCIAL_AUTH_RESPECT_X_FORWARDED_HEADERS = ENV_TOKENS.get('SOCIAL_AUTH_RESPECT_X_FORWARDED_HEADERS')
```

Comment:
@smarnach because it's python-social-auth magic :) In two words, `strategy.setting('BlaBla)` searches not only for `BlaBla`, but uses a couple of prefixes as well, last of each is `SOCIAL_AUTH_`. So this is the most flexible way to specify python-social-auth only setting: prefix the variable with `SOCIAL_AUTH_`, look up original value. In such a case, if a stronger setting is available (i,e. that should apply to entire app/instance, not only to python-social-auth) it has precedence.